words/teams.txt

```diff
@@ -1,27 +1,132 @@
+aerie
 alliance
 assembly
+bale
 band
+barrel
 batch
+bed
+bevy
+board
+brood
+building
 bunch
+business
+cackle
+camp
+cast
+catch
+cauldron
+charm
+chattering
+chime
+choir
+circle
+clan
+class
+clattering
+cloud
+clowder
 club
+cluster
+coalition
+colony
 combination
+committee
+company
 conglomerate
+congregation
+congress
+conspiracy
+convocation
+corporation
 coven
 crew
+culture
+dazzle
+descent
+doctrine
+dole
```

Comment:
slang for unemployment benefit - don't know if that's enough to get rid of it though 🤷‍♂

Reply:
better to be safe.